.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,41 @@
+name: semgrep-action
+on: [push, pull_request ]
```

Comment:
Sorry I mean the white space inside `on: [push, pull_request ]`

Reply:
Sorry should have noticed that. Updated